social/src/test/java/org/exoplatform/platform/qa/ui/social/smoke/SOC_People_Activity_CommentTestIT.java

```diff
@@ -200,6 +200,7 @@ public void test02_CommentOnYourFriendsActivity() {
     $(byId("cke_CommentTextarea" + id)).waitUntil(Condition.appears, Configuration.timeout).click();
     executeJavaScript("CKEDITOR.instances.CommentTextarea" + id + ".insertText(\"" + comment + "\")", "");
     // click on the button comment
+    Utils.pause(2000);
```

Comment:
Du the rapidity of the execution of actions with Selenide, we add waituntil. For example the save of wiki page or webcontent take 3 secondes. Selenide click on save button and search an other element defined in next step and it will not find it because the action of the add is not competed. Because of that we add waitUntil(disappears). We found many fails because of this kind of errors